modules/ROOT/pages/release-notes.adoc

```diff
@@ -54,4 +54,5 @@ Release notes for {productname} 6.0
 // 2. Prepend the inline comment markup to this element
 //    when the number of cells in the table is even.
 //a|
-//|===
+
+|===
```

Comment:
Does this also need a newline at the bottom of the page?

Reply:
Good catch. It has one now.